spec/controllers/queue_items_controller_spec.rb

```diff
@@ -102,6 +102,102 @@
       delete :destroy, id: 1
       expect(response).to redirect_to sign_in_path      
     end
+
+    it 'normalize the remaining queue items' do
+      user = Fabricate(:user)
+      session[:user_id] = user.id
+      queue_item = Fabricate(:queue_item, user: user, video: Fabricate(:video))
+      queue_item2 = Fabricate(:queue_item, user: user, video: Fabricate(:video))
+      delete :destroy, id: queue_item.id
+      expect(QueueItem.first.position).to eq(queue_item2.reload.position)
```

Comment:
Okay, after reading the tests below which are explicit like I mentioned here...it seems that this one is to verify the positioning of items that had no explicit position. I think I would still use an explicit number on the expectation even without the number in the fabricators.